Frame.cc

```diff
@@ -538,20 +549,20 @@ bool Frame::GetRasterData(std::vector<float>& image_data, CARTA::ImageBounds& bo
     bool write_lock(false);
     tbb::queuing_rw_mutex::scoped_lock lock(_cache_mutex, write_lock);
 
-    if (mean_filter) {
+    if (mean_filter && mip > 1) {
         // Perform down-sampling by calculating the mean for each MIPxMIP block
-        auto range = tbb::blocked_range2d<size_t>(0, num_rows_region, 0, row_length_region);
```

Comment:
this is not strictly related to tiled rendering, but splitting the down-sampling along one axis only helps to avoid cache misses and speeds up down-sampling around 10%.